src/pages/UserDashboardPage/util/getFilteredTasks.js

```diff
@@ -15,7 +15,8 @@ export const getFilteredTasks = (tasks = [], filter = '', projects = []) => {
     const taskStatus = task.status
     const taskType = task.taskType
     const taskPath = task.folderPath
-    const matchingStrings = [taskName, taskFolder, taskStatus, taskType, taskPath]
+    const taskAssignees = task.assigneesData.map(assignee => [assignee.name, assignee.attrib.fullName || ''].join(':')).join('::')
```

Comment:
What's the reason for all the `.join(':')).join('::')`?

Could you not just spread the name and fullName into assignees? 

`const matchingStrings = [taskName, taskFolder, taskStatus, taskType, taskPath, ...taskAssignees]`